src/main/java/slimeknights/tconstruct/gadgets/data/GadgetRecipeProvider.java

```diff
@@ -161,6 +161,20 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer) {
                        .addCriterion("has_item", hasItem(Tags.Items.DUSTS_GLOWSTONE))
                        .build(consumer, prefix(TinkerGadgets.glowBall, "gadgets/throwball/"));
 
+    // Shurikens
+    ShapedRecipeBuilder.shapedRecipe(TinkerGadgets.flintShuriken.get(), 4)
+                        .key('X', Items.FLINT)
+                        .patternLine(" X ")
+                        .patternLine("X X")
+                        .patternLine(" X ")
+                        .build(consumer, prefix(TinkerGadgets.flintShuriken, "gadgets/flint_shuriken/"));
```

Comment:
this filename mismatches with the committed file above. prefix should automatically add `flint_shuriken` for you, so I'd expect if you rerun datagen you'd get `gadgets/flint_shuriken/flint_shuriken` as the full recipe name.

Reply:
Got it. I actually think it should be `gadgets/shuriken` from looking at how the other item recipes are registered and how the current folder structures look.